chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -120,7 +120,7 @@
   group "root"
   action :create
   variables(
-    needs_block_migrate: nova[:nova][:use_shared_instance_storage] ? false : true
+    needs_block_migrate: nova.nil? || (nova[:nova][:use_shared_instance_storage] ? false : true)
```

Comment:
Stupid chef not having their old documents available ¬_¬

I have used lazy before on chef 10.x so it should work.